experiment/flake-detector/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.6
+MAINTAINER mkargaki@redhat.com
+
+RUN apk update --no-cache && \
+	apk add ca-certificates && \
+	update-ca-certificates
+
+COPY flake-detector /flake-detector
+
+ENTRYPOINT ["/flake-detector"]
```

Comment:
Any better names here? I am thinking of "sonar" or even more related to what we do: "promnar".

https://en.wikipedia.org/wiki/Sonar

Reply:
doppler is crazy enough, me likes